.ci/bumpStackReleaseVersion.groovy

```diff
@@ -99,25 +99,26 @@ def prepareArguments(Map args = [:]){
   def assign = args.get('assign', '')
   def reviewer = args.get('reviewer', '')
   // TODO: to be adapted to the 8 releases, so maybe the branch should be the one driving this dynamically.
-  def stackVersion = bumpUtils.getCurrentMinorReleaseFor7()
-  def message = """### What \n Bump stack version with the latest release. \n ### Further details \n ${stackVersion}"""
+  def stackCurrentVersion = bumpUtils.getCurrentMinorReleaseFor7()
+  def stackCurrentMinorVersion = bumpUtils.getNextMinorReleaseFor7()
```

Comment:
This is a bit odd? We get the current version by asking for the current minor but we get the current minor by asking for the next minor? Perhaps it's just the naming but this seems very confusing...

Reply:
You are right, `stackCurrentMinorVersion` is misleading :/ I'll fix it now

Reply:
```suggestion
  def stackNextMinorVersion = bumpUtils.getNextMinorReleaseFor7()
```